### PR TITLE
Let's DI inject Rootdir to avoid useless call

### PR DESCRIPTION
**Pull request by ScullWM**
Let's DI inject the RootDir (less work for the dev)

### CODE REVIEW

File: Resources/config/feed.xml [hunk 1]
```diff
@@ -15,13 +15,15 @@
         <service id="eko_feed.feed.manager" class="%eko_feed.feed.manager.class%">
             <argument type="service" id="router" />
             <argument>%eko_feed.config%</argument>
-            <argument />
```
Comment (eko) [inline Resources/config/feed.xml]: I'm not sure if we should remove this argument?

**Comment by eko** [comment]
Hi @ScullWM,

Thank you for this improvement.

File: Resources/config/feed.xml (hunk 1 above)
Comment (ScullWM) [inline Resources/config/feed.xml]: :+1: 
Seems that we can't miss the formatters. I'm going to restore this.

**Comment by eko** [comment]
Thank you @ScullWM

**Pull request merged**